cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -40,7 +40,11 @@ import (
 const (
 	DefaultCloudConfigPath = "/etc/kubernetes/cloud-config"
 
-	defaultv17AdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
+	// admission controllers for any version <= current stable release
+	// for kubeadm this is the current stable release and and 1 major release behind
+	defaultStableAdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
```

Comment:
Could we branch this as v18 and v19?
I think that would be a bit better, as the v19 chain will eventually be stable ;)